_posts/2021-07-16-1.1.0-released.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Crystal 1.1.0 is released!
+summary: The first post-1.0 release
+thumbnail: +
+author: beta-ziliani
+release_notes: true
+---
+
+We are releasing the first post-1.0 release, making efforts to keep our promise of making regular releases each 3 months, a bit delayed because of the [conference](/conference), and with special focus on keeping the language stability. Below we list the most important or interesting changes, without mentioning the several bugfixes. For details visit the [release's notes](https://github.com/crystal-lang/crystal/releases/tag/1.1.0).
+
+## Stats
+
+In this release we included [166 PRs since the 1.0.0 release](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.1.0) by 28 contributors. We thank all the effort put into improving the language and its stdlib! ❤️
+
+## Language changes
+
+1. Tuple literals now allow the use of splats:
+
+```crystal
+t = {1, 'a'}
+t2 = {true, *t} # => {true, 1, 'a'}
+typeof(t2)      # => Tuple(Bool, Int32, Char)
+
+def f1(x : {Bool, *{Int32, Char}}); end
+def f2(x : Tuple(Bool, *{Int32, Char})); end
+```
+
+Note that it also works for specifying types.
+
+2. The type of a while expression is now the union of all break expressions within the body, plus `Nil` if the condition isn't exactly the `true` literal; the while expression returns `nil` if the condition fails. The type of a break expression is computed similarly to a break inside a block or a return (`Nil` if empty, `Tuple` if multiple values).
+
+```crystal
+x = while true
+  break 1, 2  if true
+  break 3     if true
+  break 4, "" if true
+end       # => {1, 2}
+typeof(x) # => (Int32 | Tuple(Int32, Int32 | String))
+```
```

Comment:
Maybe also show a more "real" code? For example:

```crystal
# First the first non-blank line in STDIN
first_non_blank_line =
  while line = gets
    break line unless line.blank?
  end

puts first_non_blank_line
```

Reply:
Nice one!